tests/unit_tests/conftest.py

```diff
@@ -13,6 +14,22 @@
 settings.load_profile("ci")
 
 
+# This fixture is an autouse fixture to ensure that any wandb.login() calls
+# will be mocked to return a viewer server info response by default.
```

Comment:
This should be a docstring, not a comment. It should also be more concise: no need to mention that it is a fixture twice.

Is the intention just to make `wandb.login(verify=True)` pass by default? It's better to name it after the intended effect rather than its implementation:

```py
def make_wandb_login_verify_pass():
    """Patch wandb.login() to bypass network operation to verify API keys.

    Any API key that is in a valid format will be considered valid without
    attempting a connection to the server.
    """
```